recipes/libarchive/all/conanfile.py

```diff
@@ -92,7 +92,7 @@ def requirements(self):
             self.requires("libiconv/1.16")
         if self.options.with_pcreposix:
             self.requires("pcre/8.45")
-        if self.options.with_cng:
+        if self.settings.os != "Windows" and self.options.with_cng:
```

Comment:
I think that, in `package_info()`, you have to add `bcrypt` to `system_libs` if `self.settings.os == "Windows" and self.options.with_cng`.

Reply:
Would you prefer me to do:
```
self.cpp_info.libs = tools.collect_libs(self)
if self.settings.os == "Windows" and self.options.with_cng:
    self.cpp_info.system_libs.append("bcrypt")
if str(self.settings.compiler) in ["Visual Studio", "msvc"] and not self.options.shared:
    self.cpp_info.defines = ["LIBARCHIVE_STATIC"]
```
or:
```
self.cpp_info.libs = tools.collect_libs(self)
if self.settings.os == "Windows":
    if self.options.with_cng:
        self.cpp_info.system_libs.append("bcrypt")
    if str(self.settings.compiler) in ["Visual Studio", "msvc"] and not self.options.shared:
        self.cpp_info.defines = ["LIBARCHIVE_STATIC"]
```

Reply:
Would you prefer me to do:
```
self.cpp_info.libs = tools.collect_libs(self)
if self.settings.os == "Windows" and self.options.with_cng:
    self.cpp_info.system_libs.append("bcrypt")
if str(self.settings.compiler) in ["Visual Studio", "msvc"] and not self.options.shared:
    self.cpp_info.defines = ["LIBARCHIVE_STATIC"]
```
or:
```
self.cpp_info.libs = tools.collect_libs(self)
if self.settings.os == "Windows":
    if self.options.with_cng:
        self.cpp_info.system_libs.append("bcrypt")
    if str(self.settings.compiler) in ["Visual Studio", "msvc"] and not self.options.shared:
        self.cpp_info.defines = ["LIBARCHIVE_STATIC"]
```

Reply:
Ok, I did that!